docs/book/src/developer/providers/v1.0-to-v1.1.md

```diff
@@ -0,0 +1,46 @@
+# Cluster API v1.0 compared to v1.1
+
+This document provides an overview over relevant changes between ClusterAPI v1.0 and v1.1 for
```

Comment:
I tried to define what this document is meant to be, I hope I'm correct :)

I think that helps to decide what should be in the doc and what shouldn't.

I specifically don't want to create duplicate release notes.

Reply:
I've slightly aligned the structure below (Dependencies / Changes by kind) to the Kubernetes releases notes. I think it gives a nice structure, but otherwise I wouldn't try to stay consistent with them.